erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -1029,14 +1029,14 @@ def get_payment_entry(dt, dn, party_amount=None, bank_account=None, bank_amount=
 		if bank_amount:
 			received_amount = bank_amount
 		else:
-			received_amount = paid_amount * doc.conversion_rate
+			received_amount = paid_amount * (doc.get('conversion_rate') or 1)
```

Comment:
You can add the default value for `conversion_rate` in the get methods itself

```suggestion
			received_amount = paid_amount * doc.get('conversion_rate', 1)
```